### PR TITLE
fix: update support links and deprecate sentry feedback, closes #5813

### LINKED ISSUE #5813: Change link/functionality of Get support and Give feedback buttons in the wallet menu
Change the link/function:
Get support → [leather.io/contact](https://leather.io/contact)
Give feedback → [leather.io/contact](https://leather.io/contact)

Currently the page is up on leather.io/support, but we'll move to /contact instead lets make sure that is finished before merging this change.

### PR DESCRIPTION
**Pull request by pete-watters**
> Try out Leather build 21a136a — [Extension build](https://github.com/leather-io/extension/actions/runs/10767216584), [Test report](https://leather-io.github.io/playwright-reports/fix-5813-update-support-links), [Storybook](https://fix-5813-update-support-links--65982789c7e2278518f189e7.chromatic.com), [Chromatic](https://www.chromatic.com/library?appId=65982789c7e2278518f189e7&branch=fix-5813-update-support-links)

This PR updates the menu support links and deprecates Sentry feedback as per https://github.com/leather-io/extension/issues/5813

### CODE REVIEW

File: src/app/features/errors/app-error-boundary.tsx [hunk 1]
```diff
@@ -7,6 +7,8 @@ import { Box, Flex, HStack, styled } from 'leather-styles/jsx';
 import { Button, CopyIcon, Link } from '@leather.io/ui';
 import { isError } from '@leather.io/utils';
 
+import { LEATHER_SUPPORT_URL } from '@shared/constants';
```
Comment (alter-eggo) [inline src/app/features/errors/app-error-boundary.tsx]: this should be in mono no?

Comment (pete-watters) [reply src/app/features/errors/app-error-boundary.tsx]: I added it here: https://github.com/leather-io/mono/pull/425 

It seems like we have double constants in the monorepo and in `constants` here so I will clean it up

Comment (pete-watters) [reply src/app/features/errors/app-error-boundary.tsx]: @alter-eggo I've updated this, can you re-check please?

**Comment by pete-watters** [comment]
@kyranjamie can you please review this when you have a chance?

**Review by kyranjamie — APPROVED** [review]
I thought this'd been merged already.

RIP Leather User Feedback 2023–2024 🪦

**Comment by pete-watters** [comment]
> I thought this'd been merged already.
> 
> RIP Leather User Feedback 2023–2024 🪦

Thanks, yes RIP indeed. Maybe we can have it on `mobile`? Or discuss adding it back in

**Comment by markmhendrickson** [comment]
Let's just make sure we set up Slack notifications sufficiently before merging, per [this thread](https://trustmachines.slack.com/archives/C0520G8J33M/p1726748552797499).

**Comment by pete-watters** [comment]
> Let's just make sure we set up Slack notifications sufficiently before merging, per [this thread](https://trustmachines.slack.com/archives/C0520G8J33M/p1726748552797499).

Can you mark the PR as `Needs work` in case we forget. Then once Slack is setup we can re-approve + merge

**Review by markmhendrickson — CHANGES_REQUESTED** [review]
Awaiting Slack integration for Front

**Comment by markmhendrickson** [comment]
I'm closing in favor of https://github.com/leather-io/extension/pull/5897 in the spirit of testing more iterative changes to our feedback form before replacing it entirely with the new contact page.

**Pull request closed without merging**